src/lib.rs

```diff
@@ -32,7 +32,7 @@ mod endpoint;
 pub use endpoint::RemoteEndpoint;
 
 /// Error conditions that may appear during `EventManager` related operations.
-#[derive(Debug, PartialEq)]
+#[derive(Debug, Eq, PartialEq)]
```

Comment:
Can you specify in the commit message that you are deriving Eq for the `Error` enum? This is something that we will want to add to the changelog for the release and it will be easier to not miss it.

Reply:
Done, and added the changelog update too while at it